lstchain/scripts/lstchain_mc_sensitivity.py

```diff
@@ -40,31 +43,32 @@
                     help = 'path to protons DL1 file')
 parser.add_argument('--input-file-gamma-dl2-cuts', '--gd2-cuts', type = str,
                     dest = 'dl2_file_g_cuts',
-                    help = 'path to reconstructed gammas dl2 file used to caculate the sensitivity')
+                    help = 'path to reconstructed gammas dl2 file used to calculate the sensitivity')
 parser.add_argument('--input-file-proton-dl2-cuts', '--pd2-cuts', type = str,
                     dest = 'dl2_file_p_cuts',
-                    help = 'path to reconstructed protons dl2 file used to caculate the sensitivity')
+                    help = 'path to reconstructed protons dl2 file used to calculate the sensitivity')
 parser.add_argument('--input-file-gamma-dl2-sens', '--gd2-sens', type = str,
                     dest = 'dl2_file_g_sens',
-                    help = 'path to reconstructed gammas dl2 file used to caculate the optimized cuts' 
-                    'to be applied to the the sensitivity')
+                    help = 'path to reconstructed gammas dl2 file used to calculate the optimized cuts' 
+                    ' to be applied to the sensitivity')
 parser.add_argument('--input-file-proton-dl2-sens', '--pd2-sens', type = str,
                     dest = 'dl2_file_p_sens',
-                    help = 'path to reconstructed protons dl2 file used to caculate the optimized cuts'
-                    'to be applied to the the sensitivity')
+                    help = 'path to reconstructed protons dl2 file used to calculate the optimized cuts'
+                    ' to be applied to the sensitivity')
 
 args = parser.parse_args()
 
 
 def main():
-    ntelescopes_gamma = 4
-    ntelescopes_protons = 4
-    n_bins_energy = 20  #  Number of energy bins
-    n_bins_gammaness = 11  #  Number of gammaness bins
+    ntelescopes_gamma = 1
+    ntelescopes_protons = 1
+    n_bins_energy = 10  #  Number of energy bins
```

Comment:
I would prefer to leave the standard binning as it was because of the plots produced, how do the new plots look like with the binning that you are proposing?

Reply:
I think cta wants to have 10 bins per decade of energy, so something like `10**np.arange(log_e_min, log_e_max, 0.1)`

Reply:
The standard is 5 bins per decade:
https://www.cta-observatory.org/science/cta-performance/#1472563157332-1ef9e83d-426c

Reply:
ah, right, sorry, so 0.1 → 0.2

Reply:
OK so 20 bins from 0.01 to 100 TeV